Src/Newtonsoft.Json.Tests/Issues/Issue2900.cs

```diff
@@ -0,0 +1,330 @@
+﻿using System;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+#if DNXCORE50
+using Xunit;
+using Test = Xunit.FactAttribute;
+using Assert = Newtonsoft.Json.Tests.XUnitAssert;
+using TestCase = Xunit.InlineDataAttribute;
+#else
+using NUnit.Framework;
+#endif
+
+namespace Newtonsoft.Json.Tests.Issues
+{
+    public class Issue2900
+    {
+        [Test]
+        //https://github.com/JamesNK/Newtonsoft.Json/issues/2900
+        public void Override64DepthLevelOnJObjectParse()
+        {
+
+            var json = @"
+{
+  ""level1"": {
```

Comment:
You may use raw string literals to avoid `""` in json.

Reply:
I got this error when after I change to string literal,: 
  Invalid option '11' for /langversion. Use '/langversion:?' to list supported values.
Not sure if I can do that.

Reply:
You can, but nevermind, not an issue for this pr.